### ISSUE TITLE
Remove files from the project

### ISSUE THREAD

**Issue by matthewei** [metal3-io/baremetal-operator#2085]
**User Story**
I found that  there are some files that have nothing to do with this project. In order to make the project more standardized, it is recommended to remove these files.

**Detailed Description**

remove the `resources` dir

![image](https://github.com/user-attachments/assets/23acde94-0081-4f50-8318-867335de0bbd)

**Anything else you would like to add:**

[Miscellaneous information that will assist in solving the issue.]

/kind feature

**Comment by dtantsur** [comment]
Maybe we could move the keepalived container to https://github.com/metal3-io/utility-images now that we have it. I agree, having it in BMO is not very intuitive.

**Comment by tuminoid** [comment]
> Maybe we could move the keepalived container to https://github.com/metal3-io/utility-images now that we have it. I agree, having it in BMO is not very intuitive.

Agreed, and also IRSO will rip out Ironic parts out of here, also not really belonging here.

Utility images could also house ironic-client, which is more or less abandoned in its current repo?

**Comment by dtantsur** [comment]
> Utility images could also house ironic-client, which is more or less abandoned in its current repo?

It's not really abandoned, it just rarely needs changes (although we should somehow rebuild it periodically).

**Comment by tuminoid** [comment]
> > Utility images could also house ironic-client, which is more or less abandoned in its current repo?
> 
> It's not really abandoned, it just rarely needs changes (although we should somehow rebuild it periodically).

True, but with such low traffic most changes are CI related, which could be avoided if it was part of utility images. Alternatively, I could see it part of ironic repo itself as it is a debugging tool for that?

**Comment by dtantsur** [comment]
Good point, maybe having it in Ironic is the most meaningful approach.

**Comment by Rozzii** [comment]
/triage accepted
On community meeting we have agreed that ironic-client will be moved to ironic-image repo and keepalived will be moved to utilty-images.

We have agree that keepalived can be moved after the 0.9 BMO release.

/triage accepted
/help

**Comment by tuminoid** [comment]
> Good point, maybe having it in Ironic is the most meaningful approach.

Ironic-client discussion separated to its own issue and relocation is underway: https://github.com/metal3-io/ironic-client/issues/30

**Comment by Rozzii** [comment]
https://github.com/metal3-io/ironic-client/issues/30 has been resolved, thanks @tuminoid . Now the only thing left to resolve this issue is to move the keepalived when the time is right.

**Comment by mquhuy** [comment]
Are we still intending to move keepalived to utility-images

> /triage accepted On community meeting we have agreed that ironic-client will be moved to ironic-image repo and keepalived will be moved to utilty-images.
> 
> We have agree that keepalived can be moved after the 0.9 BMO release.
> 
> /triage accepted /help

Are we still intending to move keepalived to utility-images? Currently keepalived image is released and tagged together with BMO image using a pipeline inside this repo, if we move keepalived there, we need to decide if we still want keep alived as released together with BMO, or should it have its own release cycle, or something in between.

**Comment by tuminoid** [comment]
Previous change to keepalived was almost 2 years ago, so it is not often changed. The process should be very lightweight, probably just a tag.

**Comment by Rozzii** [comment]
I am doing it, I have yet again run into the topic of "why is keepalived in BMO" when I was working on https://github.com/metal3-io/ironic-image/issues/613 as I will anyhow need to modify the behavior of the keepalived scripts a little bit so I will just do it after I have done the move.

**Comment by Rozzii** [comment]
I have two PRs up related to moving the keepalived, other than merging/reviewing/tuning those what is left:

- discuss with the community what release tags to use in the future for keepalived
- testing if everything works

I would also suggest doing a release of keepalived after this PR has been merged, as part of the release we would test whether everything works as expected and I would suggest choosing a release tag that is detached from BMO.

**Comment by kashifest** [comment]
I am of the opinion of keeping the release branches as it is and then over the time we would get rid of keepalived tags eventually. We discussed this in community meeting today. The attendance was small but together with @tuminoid and @adilGhaffarDev we agreed that lets keep the keepalived release branches as it is for the sake of simplicity.

**Comment by tuminoid** [comment]
That was one of the issue we could think of why this is still open, @Rozzii to check if there is something else or can we close it. `resources` dir is gone now as originally asked.
